qutebrowser/config/configdata.py

```diff
@@ -1125,6 +1124,14 @@ def data(readonly=False):
              SettingValue(typ.QssColor(), 'black'),
              "Background color of the statusbar."),
 
+            ('statusbar.fg.private',
+             SettingValue(typ.QssColor(), '${statusbar.fg}'),
+             "Foreground color of the statusbar in private browsing mode."),
+
+            ('statusbar.bg.private',
+             SettingValue(typ.QssColor(), 'grey'),  # FIXME darker color
```

Comment:
Well, then the question is whether we need `statusbar.bg.insert.private`, and the same for `caret` and `caret-selection` as well :wink:

(probably not - I agree having the color change when entering commands is slightly annoying)